src/app/services/categories-api.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, map } from 'rxjs';
+import { Category } from '../shared/interfaces/category';
+import { Article } from '../shared/interfaces/article';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CategoriesApiService {
+  private apiUrl: string = "http://json.helaine.eu/api/";
+
+  constructor(private http: HttpClient) {}
+
+  getCategories(): Observable<Category[]> {
+    const url = `${this.apiUrl}/categories.php`;
+    return this.http.get<Category[]>(url);
+  }
+
+  getArticlesByCategorySlug(slug: string): Observable<Article[]> {
```

Comment:
It's normal, for now we don't have enough articles for each category for it to be worth using.
So for now the method gets every article.
Next sprint it will be used